### ISSUE TITLE
Ember angle bracket syntax support in handlebars

### ISSUE THREAD

**Issue by rreckonerr** [beautify-web/js-beautify#1743]
# Input
The code looked like this before beautification:
```handlebars
<SomeComponent as |sc|>
    ...some content...
</SomeComponent>
```

# Expected Output
The code should have looked like this after beautification:
```handlebars
<SomeComponent as |sc|>
    ...some content...
</SomeComponent>
```

# Actual Output
The  code actually looked like this after beautification:
```handlebars
  <SomeComponent
    as
    |sc|
  >
    ...some content...
  </SomeComponent>
```

# Steps to Reproduce

## Environment
OS: MacOS

**Comment by bitwiseman** [comment]
Can you link to the spec or docs for this?  We'll also need a bit more context - is this in an html file or some other kind?

**Comment by rreckonerr** [comment]
@bitwiseman check out documentation for block parameters [here](https://guides.emberjs.com/release/components/block-content/). It happens in handlebars files.

**Comment by rreckonerr** [comment]
@bitwiseman Hello again! Do you have everything needed to see what's the issue about? I can provide you with extra info if there's still something unclear.

**Comment by bitwiseman** [comment]
Yes, thanks that should do.  I don't have time to work on this myself, but when someone has time it's here. 
Thanks.
